### ISSUE TITLE
Participant actions glitch: *Remove participant* shown instead of *Send call notification*

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/spreed#11579]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1. Start a silent call
2. Go to the participants list
3. Open the actions menu
4. Wait for *Send call notification* action to be available

### Expected behaviour

The usual menu

### Actual behaviour

*Remove participant* shows twice

![Bildschirmfoto vom 2024-02-08 14-00-55](https://github.com/nextcloud/spreed/assets/1374172/a6ece185-05b6-414a-9579-7de6d4f86cd7)

## Talk app

**Talk app version:** (see apps admin page: `/index.php/settings/apps`)

**Custom Signaling server configured:** yes/no and version (see additional admin settings: `/index.php/index.php/settings/admin/talk#signaling_server`)

**Custom TURN server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/talk#turn_server`)

**Custom STUN server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/talk#stun_server`)

## Browser

**Microphone available:** yes/no

**Camera available:** yes/no

**Operating system:** Windows/Ubuntu/...

**Browser name:** Firefox/Chrome/...

**Browser version:** 85/96/...

### Browser log

<details>
```
Insert your browser log here, this could for example include:
a) The javascript console log
b) The network log
c) ...
```

</details>

## Server configuration

**Operating system**: Ubuntu/RedHat/...

**Web server:** Apache/Nginx

**Database:** MySQL/Maria/SQLite/PostgreSQL

**PHP version:** 8.0/8.1/8.2

**Nextcloud Version:** (see admin page)

**List of activated apps:**

<details>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your server installation folder
```
</details>

**Nextcloud configuration:**

<details>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder
```
</details>

### Server log (data/nextcloud.log)
<details>

```
Insert your server log here
```
</details>

**Comment by DorraJaouad** [comment]
Fixed with #11544 (probably, as it was hard to reproduce)

**Comment by ChristophWurst** [comment]
Duplicate of https://github.com/nextcloud/spreed/issues/11512
